lib/ansible/modules/network/radware/vdirect_runnable.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/python
+#  -*- coding: utf-8 -*-
+#
+# Copyright 2017 Radware LTD.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+module: vdirect_runnable
+author: Evgeny Fedoruk @ Radware LTD (@evgenyfedoruk)
+short_description: Runs templates and workflow actions in Radware vDirect server
+description:
+    - Runs configuration templates, creates workflows and runs workflow actions in Radware vDirect server.
+notes:
+    - Requires the Radware vdirect-client Python package on the host. This is as easy as
+      C(pip install vdirect-client)
+version_added: "2.5"
+options:
+  vdirect_ip:
+    description:
+     - Primary vDirect server IP address, may be set as VDIRECT_IP environment variable.
```

Comment:
`C(VDIRECT_IP)`
http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options

Reply:
Thanks,
will fix it on next commit.

I prefix all parameters with "vdirect_" to keep it similar to environment variables that may be used instead of specifying the parameter. vdirect is the back-end system the module communicates with for all operations.